index.html

```diff
@@ -5269,23 +5269,24 @@ <h2>
     </section>
     <section id="conformance">
       <p>
-        Conformance checkers that claim support for checking of ARIA in HTML,
+        Conformance checkers that claim support for checking ARIA in HTML,
         MUST implement checks for the document conformance requirements for use
-        of the ARIA `role` and `aria-*` attributes on HTML elements as defined
-        in this specification.
+        of the ARIA `role` and `aria-*` attributes on [=HTML elements=] as
+        defined in this specification.
       </p>
       <p>
-        Documents MUST NOT use any `role`
-        values with elements in the [[[#docconformance]]] table, other than the
-        corresponding role value (if any) as listed for that element in the
-        third column, other than those indicated in the second column (whose
-        explicit use in markup is NOT RECOMMENDED).
+        A conforming document MUST NOT contain any elements with author defined `role` or `aria-*` attributes with values other than the
+        values which have been indicated authors MAY use for each ARIA
+        attribute, on each [=HTML element=], in [[[#docconformance]]].
```

Comment:
Just trying to delete some words... :)

```suggestion
        A conforming document MUST NOT contain any elements with author defined `role` or `aria-*` attributes with values other than those
        which have been indicated authors MAY use on each [=HTML element=] in [[[#docconformance]]].
```